PWGEM/Tasks/phosCalibration.cxx

```diff
@@ -67,14 +67,16 @@ struct phosCalibration {
     bool fBad;
   };
 
-  Configurable<double> mMinCellAmplitude{"minCellAmplitude", 0.3, "Minimum cell amplitude for histograms."};
+  Configurable<int> flagCCDB{"flagCCDB", 1, "Flag for choosing PATH to L1Phase data"};
+  Configurable<double> mMinCellAmplitude{"minCellAmplitude", 0.3, "Minimum cell amplitude for histograms"};
   Configurable<double> mCellTimeMinE{"minCellTimeAmp", 100, "Minimum cell amplitude for time histograms (ADC)"};
   Configurable<double> mMinCellTimeMain{"minCellTimeMain", -50.e-9, "Min. cell time of main bunch selection"};
   Configurable<double> mMaxCellTimeMain{"maxCellTimeMain", 100.e-9, "Max. cell time of main bunch selection"};
   Configurable<int> mMixedEvents{"mixedEvents", 10, "number of events to mix"};
   Configurable<bool> mSkipL1phase{"skipL1phase", false, "do not correct L1 phase from CCDB"};
   Configurable<std::string> mBadMapPath{"badmapPath", "alien:///alice/cern.ch/user/p/prsnko/Calib/BadMap/snapshot.root", "path to BadMap snapshot"};
   Configurable<std::string> mCalibPath{"calibPath", "alien:///alice/cern.ch/user/p/prsnko/Calib/CalibParams/snapshot.root", "path to Calibration snapshot"};
+  Configurable<std::string> mL1PhasePath{"L1PhasePath", "alien:///alice/cern.ch/user/d/daveryan/Calib/L1Phase/", "path to L1Phase snapshot"};
```

Comment:
NO alien files should be accessed directly from code.

Reply:
Dear @costing, this option is added for final check of calibration before committing to CCDB. Which option would you suggest instead to store transient files used either for one of iterations of calibration or for check of the produced calibration? Thanks, Dmitri

Reply:
Hi Dmitri,

For this purpose the personal space in CCDB could be used. You can upload objects yourself to for example `https://alice-ccdb.cern.ch/Users/d/daveryan/`, via the API or using this web interface: https://alimonitor.cern.ch/ccdb/upload.jsp . That allows testing the full ccdb functionality with the new object and when it's proven to work correctly it's a simple operation to copy or move it to production.

Cheers,

.costin